api/share/utils.py

```diff
@@ -241,8 +241,7 @@ def format_registration(registration, *args, **kwargs):
             'description': registration.description or '',
             'is_deleted': not registration.is_public or registration.is_deleted or is_qa,
             'date_published': registration.registered_date.isoformat() if registration.registered_date else None,
-            'registration_type': registration.registered_schema.first().name if registration.registered_schema else None,
-            'withdrawn': registration.is_retracted,
```

Comment:
I was surprised I had to remove this line but I got a the following error `Additional properties are ' "not allowed ('withdrawn' was unexpected) at "'/@graph/0","source":{"pointer":"/data/attributes/data"},"status":"400"}]` which seem clear.  Line 244 is a single logic error.

Reply:
Discussed in flowdock about the necessity of the registered schema existence check, but decided better safe than sorry.